src/OpenSearch/Endpoints/Search.php

```diff
@@ -78,7 +78,8 @@ public function getParamWhitelist(): array
             'batched_reduce_size',
             'max_concurrent_shard_requests',
             'pre_filter_shard_size',
-            'rest_total_hits_as_int'
+            'rest_total_hits_as_int',
+            'highlight'
```

Comment:
Hi @prattcmp, it looks like highlight isn't a valid parameter for the Search API. I tested it. Search doesn't include the highlight parameter [here](https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/rest/action/search/RestSearchAction.java).

If you think there's something missing, feel free to contribute! You can also consider contributing to the opensearch API specification [here](https://github.com/opensearch-project/opensearch-api-specification/blob/main/spec/namespaces/_core.yaml#L637) and adding any missing parameters. Thanks!